resources/leiningen/new/kraken_http_api/service.clj

```diff
@@ -0,0 +1,35 @@
+(ns {{name}}.service
+  (:require [io.pedestal.http :as bootstrap]
+            [io.pedestal.http.route :as route]
+            [io.pedestal.http.route.definition :refer [defroutes]]
+            [io.pedestal.interceptor :refer [interceptor]]
+            [ring.util.response :as ring-resp]
+            [turbovote.resource-config :refer [config]]
+            [pedestal-toolbox.params :refer :all]
+            [pedestal-toolbox.content-negotiation :refer :all]
+            [kehaar.core :as k]
+            [clojure.core.async :refer [chan go alt! timeout]]))
+
+(def ping
```

Comment:
Just a thought, but should the ping make use of the ok queue? I mean, it's of marginal value to know the http-api is up, it's more important to know that it's up and able to talk with the back end. Or is there a better way to do that?